src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,31 @@
-export const Tabs = () => {};
+export const Tabs = ({ tabs, selectedTab, onTabSelected }) => (
+  <div data-cy="TabsComponent">
+    <div className="tabs is-boxed">
+      <ul>
+        {tabs.map(tab => {
+          const isActive = tab.title === selectedTab.title;
```

Comment:
```suggestion
          const isActive = tab.id === selectedTab.id;
```
better to compare using the id instead of the title